from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(800, 600)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setObjectName("gridLayout")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setObjectName("label")
        self.gridLayout.addWidget(self.label, 0, 0, 1, 1)
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setObjectName("label_2")
        self.gridLayout.addWidget(self.label_2, 0, 1, 1, 1)
        self.textEdit = QtWidgets.QTextEdit(self.centralwidget)
        self.textEdit.setObjectName("textEdit")
        self.gridLayout.addWidget(self.textEdit, 1, 0, 1, 1)
        self.textEdit_2 = QtWidgets.QTextEdit(self.centralwidget)
        self.textEdit_2.setObjectName("textEdit_2")
        self.gridLayout.addWidget(self.textEdit_2, 1, 1, 1, 1)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 25))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtWidgets.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuOpen = QtWidgets.QMenu(self.menuFile)
        self.menuOpen.setObjectName("menuOpen")
        self.menuSave = QtWidgets.QMenu(self.menuFile)
        self.menuSave.setObjectName("menuSave")
        self.menuNew = QtWidgets.QMenu(self.menuFile)
        self.menuNew.setObjectName("menuNew")
        self.menuConvert_2 = QtWidgets.QMenu(self.menubar)
        self.menuConvert_2.setObjectName("menuConvert_2")
        self.menuTo_Ingush = QtWidgets.QMenu(self.menuConvert_2)
        self.menuTo_Ingush.setObjectName("menuTo_Ingush")
        self.menuTo_Avar = QtWidgets.QMenu(self.menuConvert_2)
        self.menuTo_Avar.setObjectName("menuTo_Avar")
        self.menuTo_Chechen = QtWidgets.QMenu(self.menuConvert_2)
        self.menuTo_Chechen.setObjectName("menuTo_Chechen")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionTo_Ingush = QtWidgets.QAction(MainWindow)
        self.actionTo_Ingush.setObjectName("actionTo_Ingush")
        self.actionTo_Avar = QtWidgets.QAction(MainWindow)
        self.actionTo_Avar.setObjectName("actionTo_Avar")
        self.actionTo_Chechen = QtWidgets.QAction(MainWindow)
        self.actionTo_Chechen.setObjectName("actionTo_Chechen")
        self.actionExit = QtWidgets.QAction(MainWindow)
        self.actionExit.setObjectName("actionExit")
        self.actionWith_original = QtWidgets.QAction(MainWindow)
        self.actionWith_original.setObjectName("actionWith_original")
        self.actionIngush_with_original = QtWidgets.QAction(MainWindow)
        self.actionIngush_with_original.setObjectName("actionIngush_with_original")
        self.actionIngush_without_original = QtWidgets.QAction(MainWindow)
        self.actionIngush_without_original.setObjectName("actionIngush_without_original")
        self.actionAvar_with_original = QtWidgets.QAction(MainWindow)
        self.actionAvar_with_original.setObjectName("actionAvar_with_original")
        self.actionAvar_without_original = QtWidgets.QAction(MainWindow)
        self.actionAvar_without_original.setObjectName("actionAvar_without_original")
        self.actionChechen_with_original = QtWidgets.QAction(MainWindow)
        self.actionChechen_with_original.setObjectName("actionChechen_with_original")
        self.actionChechen_without_original = QtWidgets.QAction(MainWindow)
        self.actionChechen_without_original.setObjectName("actionChechen_without_original")
        self.actionOpen_original = QtWidgets.QAction(MainWindow)
        self.actionOpen_original.setObjectName("actionOpen_original")
        self.actionOpen_converted = QtWidgets.QAction(MainWindow)
        self.actionOpen_converted.setObjectName("actionOpen_converted")
        self.actionSave_original = QtWidgets.QAction(MainWindow)
        self.actionSave_original.setObjectName("actionSave_original")
        self.actionSave_converted = QtWidgets.QAction(MainWindow)
        self.actionSave_converted.setObjectName("actionSave_converted")
        self.actionNew_original = QtWidgets.QAction(MainWindow)
        self.actionNew_original.setObjectName("actionNew_original")
        self.actionNew_converted = QtWidgets.QAction(MainWindow)
        self.actionNew_converted.setObjectName("actionNew_converted")
        self.menuOpen.addAction(self.actionOpen_original)
        self.menuOpen.addAction(self.actionOpen_converted)
        self.menuSave.addAction(self.actionSave_original)
        self.menuSave.addAction(self.actionSave_converted)
        self.menuNew.addAction(self.actionNew_original)
        self.menuNew.addAction(self.actionNew_converted)
        self.menuFile.addAction(self.menuNew.menuAction())
        self.menuFile.addAction(self.menuOpen.menuAction())
        self.menuFile.addAction(self.menuSave.menuAction())
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionExit)
        self.menuTo_Ingush.addAction(self.actionIngush_with_original)
        self.menuTo_Ingush.addAction(self.actionIngush_without_original)
        self.menuTo_Avar.addAction(self.actionAvar_with_original)
        self.menuTo_Avar.addAction(self.actionAvar_without_original)
        self.menuTo_Chechen.addAction(self.actionChechen_with_original)
        self.menuTo_Chechen.addAction(self.actionChechen_without_original)
        self.menuConvert_2.addAction(self.menuTo_Ingush.menuAction())
        self.menuConvert_2.addAction(self.menuTo_Avar.menuAction())
        self.menuConvert_2.addAction(self.menuTo_Chechen.menuAction())
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuConvert_2.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Converter in Python with PyQt5"))
        self.label.setText(_translate("MainWindow", "<b>Original list</b>"))
        self.label_2.setText(_translate("MainWindow", "<b>Converted list</b>"))
        self.menuFile.setTitle(_translate("MainWindow", "File"))
        self.menuOpen.setTitle(_translate("MainWindow", "Open"))
        self.menuSave.setTitle(_translate("MainWindow", "Save"))
        self.menuNew.setTitle(_translate("MainWindow", "New"))
        self.menuConvert_2.setTitle(_translate("MainWindow", "Convert"))
        self.menuTo_Ingush.setTitle(_translate("MainWindow", "To Ingush"))
        self.menuTo_Avar.setTitle(_translate("MainWindow", "To Avar"))
        self.menuTo_Chechen.setTitle(_translate("MainWindow", "To Chechen"))
        self.actionTo_Ingush.setText(_translate("MainWindow", "To Ingush"))
        self.actionTo_Avar.setText(_translate("MainWindow", "To Avar"))
        self.actionTo_Chechen.setText(_translate("MainWindow", "To Chechen"))
        self.actionExit.setText(_translate("MainWindow", "Exit"))
        self.actionWith_original.setText(_translate("MainWindow", "With original"))
        self.actionIngush_with_original.setText(_translate("MainWindow", "Ingush with original"))
        self.actionIngush_without_original.setText(_translate("MainWindow", "Ingush without original"))
        self.actionAvar_with_original.setText(_translate("MainWindow", "Avar with original"))
        self.actionAvar_without_original.setText(_translate("MainWindow", "Avar without original"))
        self.actionChechen_with_original.setText(_translate("MainWindow", "Chechen with original"))
        self.actionChechen_without_original.setText(_translate("MainWindow", "Chechen without original"))
        self.actionOpen_original.setText(_translate("MainWindow", "Open original"))
        self.actionOpen_converted.setText(_translate("MainWindow", "Open converted"))
        self.actionSave_original.setText(_translate("MainWindow", "Save original"))
        self.actionSave_converted.setText(_translate("MainWindow", "Save converted"))
        self.actionNew_original.setText(_translate("MainWindow", "New original"))
        self.actionNew_converted.setText(_translate("MainWindow", "New converted"))

